remotespark/wrapperkernel/codetransformers.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) 2015  aggftw@gmail.com
+# Distributed under the terms of the Modified BSD License.
+
+from remotespark.utils.constants import Constants
+
+
+class UserCodeTransformerBase(object):
+    def __init__(self, subcommand):
+        self.name = subcommand
+
+    def get_code_to_execute(self, session_started, connection_string, force, output_var, command):
+        raise NotImplementedError()
+
+
+class NotSupportedTransformer(UserCodeTransformerBase):
+    def get_code_to_execute(self, session_started, connection_string, force, output_var, command):
+        error_to_show = "Magic '{}' not supported.".format(self.name)
+        code_to_run = ""
+        begin_action = Constants.do_nothing_action
+        end_action = Constants.do_nothing_action
+        deletes_session = False
+
+        return code_to_run, error_to_show, begin_action, end_action, deletes_session
+
+
+class ConfigTransformer(UserCodeTransformerBase):
+    def get_code_to_execute(self, session_started, connection_string, force, output_var, command):
```

Comment:
As with the above comment, it might be fruitful in the future to refactor this so that we have a class which represents a "command" which has fields like "connection_string", "force", "command", etc.  This is going to be more scalable and simple than passing around a bunch of tokens.

We're OK for now but this should really be refactored in the near future because this only gets more complicated as we add more functionality.